teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -548,6 +549,28 @@ interface Change {
     val date: Date
 }
 
+interface ChangeFiles {
+    val count: Int
```

Comment:
What is meaning of `count`? Is it always equal to `files.size`? Do we really need it given that information about files is already in memory? If we get rid of `count`, we'll be able to remove `ChangeFiles` interface and return `List<ChangeFile>` from `Change` directly, this will simplify API.